package.json

```diff
@@ -31,15 +31,18 @@
     "eslint-plugin-jest": "^24.1.3",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-react": "^7.22.0",
-    "eslint-plugin-react-hooks": "^4.2.0"
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "dotenv-cli": "^4.0.0",
+    "gh-pages": "^3.1.0"
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "dotenv -e .env.production react-scripts build",
```

Comment:
Turns out we need a separate utility (`dotenv`) to set different `PUBLIC_URL` per environment. We set our config in a separate file (`.env.production`) and pass this file to `dotenv` in build step.